docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     privileged: true
     devices:
       - /dev/bus/usb:/dev/bus/usb
+      - /dev/apex_0:/dev/apex_0
```

Comment:
Does this prevent the container from starting on device types where this is not present? I will have to test on NUC and Raspberry Pi unless you already have?

Reply:
Good thought, I haven't tried this!

Reply:
Since we are running as privileged (in order to mount usb storage) we shouldn't really need to expose any devices, do we? I mean if we were being specific about adding caps it would matter but with privileged I think we get all devices for free.

Reply:
@klutchell yep you are right, I've removed the devices section completely and am still using the Coral for inferencing with no problems.